src/Symfony/Component/Validator/Constraints/AllValidator.php

```diff
@@ -44,12 +44,12 @@ public function validate($value, Constraint $constraint)
             $validator = $context->getValidator()->inContext($context);
 
             foreach ($value as $key => $element) {
-                $validator->atPath('['.$key.']')->validate($element, $constraint->constraints);
+                $validator->atPath('['.$key.']')->validate($element, $constraint->constraints, $constraint->groups);
```

Comment:
Hi, @xabbuh 

I try to use the context but the test fail, it's the only way I find to make it work:

``` 
1) Symfony\Component\Validator\Tests\Validator\LegacyValidator2Dot5ApiTest::testValidateMultipleGroupsForAllConstraint
Failed asserting that actual size 1 matches expected size 2.

symfony/src/Symfony/Component/Validator/Tests/Validator/AbstractValidatorTest.php:1129

2) Symfony\Component\Validator\Tests\Validator\LegacyValidatorLegacyApiTest::testValidateMultipleGroupsForAllConstraint
Failed asserting that actual size 1 matches expected size 2.

symfony/src/Symfony/Component/Validator/Tests/Validator/AbstractValidatorTest.php:1129

3) Symfony\Component\Validator\Tests\Validator\RecursiveValidator2Dot5ApiTest::testValidateMultipleGroupsForAllConstraint
Failed asserting that actual size 1 matches expected size 2.
```

Do you have another idea ?